rdt-open-reader/src/testCovidIndonesia/java/io/ona/rdt/widget/CovidDatePickerFactoryTest.java

```diff
@@ -49,6 +49,12 @@ public void testMinAllowedDateValidatorIsAdded() throws Exception {
         Assert.assertEquals(actualMinAllowedDateStr, expectedMinAllowedDateStr);
     }
 
+    @Test
+    public void testCatchDatePickerException() throws Exception {
+        MaterialEditText materialEditText = getDatePickerEditText(null);
+        Assert.assertFalse(materialEditText.hasValidators());
+    }
```

Comment:
We can use the [expected](https://stackoverflow.com/a/156528/9782187) exception option if we want to verify that an exception is thrown.

Reply:
Expected exception not working because we already catch the exception in attachLayout method